app/src/main/java/ip/team13/petowner/ui/leaderboard/LeaderboardViewModel.kt

```diff
@@ -1,6 +1,29 @@
 package ip.team13.petowner.ui.leaderboard
 
-import androidx.lifecycle.ViewModel
+import androidx.databinding.Bindable
+import androidx.databinding.ObservableField
+import ip.team13.petowner.core.BaseViewModel
+import ip.team13.petowner.data.domain.LeaderboardEntry
+import ip.team13.petowner.data.domain.LeaderboardType
+import ip.team13.petowner.data.repository.LeaderboardRepository
 
-class LeaderboardViewModel : ViewModel() {
+class LeaderboardViewModel(
+    private val leaderboardRepository: LeaderboardRepository
+) : BaseViewModel() {
+
+    val leaderboardType = ObservableField(LeaderboardType.FREE)
+
+    val person1 = ObservableField<LeaderboardEntry>()
+    val person2 = ObservableField<LeaderboardEntry>()
+    val person3 = ObservableField<LeaderboardEntry>()
```

Comment:
Why don't we use the actual type for each of these 4 fields and add the @get:Bindable annotation? It feels extra having to call `get()` and `set()` everytime we use them

Example:
```
@get:Bindable
val person1: LeaderboardEntry?
    get() = items[0]
```

Reply:
Sure, will change